openassessment/xblock/test/test_peer.py

```diff
@@ -286,7 +286,6 @@ def test_turbo_grading(self, xblock):
         self.assertIsNotNone(peer_response)
         self.assertNotIn(submission["answer"]["parts"][0]["text"].encode('utf-8'), peer_response.body)
         self.assertNotIn(submission["answer"]["parts"][1]["text"].encode('utf-8'), peer_response.body)
-        self.assertIn("Peer Assessments Complete", peer_response.body)
```

Comment:
Why was this test change included? I'm not seeing how it connects to the rest of this PR.